### PR TITLE
Add Gelma's typos from "j" to "m"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -32690,6 +32694,8 @@ laguagues->languages
 laguange->language
 laguanges->languages
 laiter->later
+lamba->lambda, lambs,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:32697]: Also singular `lamb`, as well as any of `lama`, `mamba`, `samba` and even `lambada`?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -32690,6 +32694,8 @@ laguagues->languages
...
+lambas->lambdas, lambs,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:32698]: Also `lambast`/`lambaste`, as well as `lamas`, `mambas`?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -32945,6 +32951,8 @@ legth->length
 legths->lengths
 leibnitz->leibniz
 leightweight->lightweight
+lement->element, clement, lament,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:32954]: Any typos of `cement` and `dement` out there?

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -33230,6 +33239,10 @@ linaer->linear
 linaerly->linearly
 linar->linear
 linarly->linearly
+lincense->license
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:33242]: Perhaps `incense` too?

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -33230,6 +33239,10 @@ linaer->linear
...
+lincensed->licensed
+lincenses->licenses
+lincensing->licensing
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:33245]: Also add typos of `licence` and it verbal forms?

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -33445,6 +33459,12 @@ loclaizes->localizes
 loclaizing->localizing
 loclaly->locally
 loclas->locals
+locte->locate
+locted->located, locked,
+loctes->locates
+locting->locating, locking,
+loction->location, lotion, locution,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:33466]: Perhaps `lection` too?

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -34507,6 +34602,7 @@ metod->method
 metodologies->methodologies
 metodology->methodology
 metods->methods
+metrices->metrics, matrices,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:34605]: Add singular `metrix->metric, matrix,`?

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:32697]: I'll add `lamb`. I couldn't find the other typos.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:32698]: I'll add `lambast`. I couldn't find the other typos.

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:32954]: I couldn't find these typos.

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:33242]: I couldn't find this typo.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**